explicit match statement

Grompiler opened the pull request:
Ok ca change rien dans ce cas, mais quand on match on essaye de toujours expliciter tous les choix. Faire ca permet de s'aider du compilo pour qu'il nous avertisse en cas d'erreur ou d'oubli de branche a matcher.
Utiliser `_` ne lui permet pas de faire ses checks rigoureux.

fabien14 reviewed (approved):
J'avais bien en tete ce que tu viens de m'expliquer mais je ne savais pas quoi mettre a la place du _

Commit:
explicit match statement

Ok ca change rien dans ce cas, mais quand on match on essaye de toujours expliciter tous les choix.
Faire ca permet de s'aider du compilo pour qu'il nous avertisse en cas
d'erreur ou d'oublie de branche a matcher.
Utiliser `_` ne lui permet pas de faire ses checks rigoureux.
